apps/backend/src/db/migrations/20250805171700_super_admin_seed.ts

```diff
@@ -0,0 +1,47 @@
+import { type Knex } from "knex";
+
+const TableName = {
+	GROUPS: "groups",
+	USERS: "users",
+} as const;
+
+const GROUP_KEY = "admins";
+
+const ColumnName = {
+	EMAIL: "email",
+	ID: "id",
+	KEY: "key",
+} as const;
+
+type IdRecord = { id: number };
+
+const SUPERADMIN_EMAIL = "superadmin@mail.com";
```

Comment:
let's use smart-scapes as email domain, because usually you create an email from the company domain for such purposes
```suggestion
const SUPERADMIN_EMAIL = "superadmin@smart-scapes.com";
```